stable_baselines3/common/utils.py

```diff
@@ -110,3 +112,19 @@ def get_device(device: Union[th.device, str] = 'auto') -> th.device:
         return th.device('cpu')
 
     return device
+
+
+def get_latest_run_id(log_path, log_name):
+    """
+    Returns the latest run number for the given log name and log path,
+    by finding the greatest number in the directories.
+
+    :return: (int) latest run number
+    """
+    max_run_id = 0
+    for path in glob.glob("{}/{}_[0-9]*".format(log_path, log_name)):
```

Comment:
please use f-string

Reply:
(and add typing)

Reply:
Regarding typing, I see in the code sometimes `Optional[str]` and sometimes `Optional[str] = None`. What would be the differences between these and `str = None`?

Reply:
`str = None` is not correct, `Optional` means that the variable can be `None` (but `None` is not a string)

`Optional[str] = None` is when you give a default value